test/functional/p2p_mutated_blocks.py

```
@@ -0,0 +1,96 @@
+#!/usr/bin/env python3
```

Ruff (0.12.2)

Shebang is present but file is not executable

(EXE001)